azext_edge/edge/providers/check/mq.py

```diff
@@ -375,27 +493,36 @@ def evaluate_brokers(
                 frontend_replicas: Optional[int] = broker_cardinality.get("frontend", {}).get("replicas")
 
                 if backend_partition_count and backend_partition_count >= 1:
-                    backend_chain_count_colored = f"[green]Actual {backend_partition_count}[/green]."
+                    backend_chain_count_colored = (
+                        f"{colorize_string(color='green', value=f'Actual {backend_partition_count}')}."
+                    )
                 else:
-                    backend_chain_count_colored = f"[red]Actual {backend_partition_count}[/red]."
+                    backend_chain_count_colored = (
+                        f"{colorize_string(color='red', value=f'Actual {backend_partition_count}')}."
+                    )
                     broker_eval_status = CheckTaskStatus.error.value
 
                 if backend_redundancy and backend_redundancy >= 1:
-                    backend_replicas_colored = f"[green]Actual {backend_redundancy}[/green]."
+                    backend_replicas_colored = (
+                        f"{colorize_string(color='green', value=f'Actual {backend_redundancy}')}."
+                    )
                 else:
-                    backend_replicas_colored = f"[red]Actual {backend_redundancy}[/red]."
+                    backend_replicas_colored = f"{colorize_string(color='red', value=f'Actual {backend_redundancy}')}."
                     broker_eval_status = CheckTaskStatus.error.value
 
                 if backend_workers and backend_workers >= 1:
-                    backend_workers_colored = f"[green]Actual {backend_workers}[/green]."
+                    backend_workers_colored = f"{colorize_string(color='green', value=f'Actual {backend_workers}')}."
                 else:
-                    backend_workers_colored = f"[red]Actual {backend_workers}[/red]."
+                    backend_workers_colored = f"{colorize_string(color='red', value=f'Actual {backend_workers}')}."
```

Comment:
np! your suggestion make sense, yeah I'm trying to clean up all strings use the colorize_string you introduced earlier(feels like that's better), I can address your suggestion in current pr since it's not in a rush